### DISCUSSION TITLE
Self hosted hardware sizing requirements

### DISCUSSION THREAD

**Discussion by IO-igiannako** [plausible/analytics#2638] [Self-Hosted Support] [answered]
I was look for a guide on how to properly plan on sizing in terms of CPU, Memory and Disk size for an average or 10 sites having about 1M pages views per month, each. Is there a guide or a requirements blueprint I can use to analyse and plan for own hosting?

**Comment by Tony-MF** [comment, marked as answer]
Hi, 

I'm running a fresh instance with 12 sites which have about 2.5M users/**day** in total on a 48 GB / 16 vCPUs / NVMe server and the server is bored. In the peak hours (same for the all sites) the load was 3.5 and CPU usage was less than 20%.

**Comment by IO-igiannako** [comment]
Thanks tons @Tony-MF, I appreciate it! Any size estimation on disk utilisation?
What seems like a realistic figure in Clickhouse size per month for your instance? Do you happen to have a number for directory size?

Thanks in advance, helpful and enlightening response!

**Reply by Tony-MF** [reply]
I think someone else will answer this better. I don't have enough data to calculate the growth rate.

I'm migrating to Plausible from GA for the last 2-3 days and right now I have there 24 sites where half of them have imported data from GA and the docker volume has 1.3GB only.

**Reply by IO-igiannako** [reply]
Thanks for taking the time to write back. Don't hesitate to revisit this discussion in case you have something to share in due time.
